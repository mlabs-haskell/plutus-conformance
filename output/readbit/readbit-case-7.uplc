[ [ (builtin readBit) (con bytestring #f4) ] (con integer 2) ]