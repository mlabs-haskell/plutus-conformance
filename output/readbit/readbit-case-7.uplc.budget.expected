{"exBudgetCPU":100000080100,"exBudgetMemory":100000000600}